package.json

```diff
@@ -38,6 +38,7 @@
     "packages:clean": "lerna run clean",
     "precommit": "yarn run lint-staged",
     "prettier:check": "prettier --check --list-different=false --loglevel=warn \"**/*.{ts,tsx,scss,md,mdx}\"",
+    "prettier:checkDocs": "prettier --check --list-different=false --loglevel=warn \"docs/**.md\" \"packages/**.{ts,tsx,scss,md,mdx}\"",
```

Comment:
Hm I am not sure, I think the default `prettier:check` also checks for those things.
Also, there are some `.md` files there as well. Shouldn't we also do in this step?
Otherwise we can remove it.

Reply:
It depends on if these packages/files are triggered part of the docs-pipeline conditions, if not I think we should leave them out,

Reply:
Yes they are here: https://github.com/grafana/grafana/blob/main/.drone.yml#L462

Reply:
Okay than it makes sense I guess